HSP2/main.py

```diff
@@ -14,19 +14,35 @@
 
 import os
 import importlib
+from datetime import datetime as dt
+from collections import defaultdict
+
 import h5py
 import pandas as pd
 import numpy as np
-from datetime import datetime as dt
-from collections import defaultdict
+import mando
+from mando.rst_text_formatter import RSTHelpFormatter
 
 import HSP2
 
-
-def run(hdfname, saveall=False, reloadkeys= False):
-    ''' runs main HSP2 program
-     saveall    - (optional) saves all calculated data ignoring SAVE tables
-     reloadkeys - (optional) regenerates keys, used after adding new modules'''
+@mando.command(formatter_class=RSTHelpFormatter, doctype='numpy')
+def run(hdfname,
+        saveall=False,
+        reloadkeys=False):
+    '''Runs main HSP2 program.
+
+    Parameters
+    ----------
+    hdfname: str
+        HDF5 filename for both input and
+        output.
+    saveall
```

Comment:
The original line for saveall indicates this is an optional argument.  I do not expect users to change this in general.  Having a default argument does not automatically carry this information.  I would prefer an indication this is a optional.

Reply:
Easy fix - will do.